package.json

```diff
@@ -46,5 +52,21 @@
   ],
   "engines": {
     "node": ">= 12.0.0"
+  },
+  "nyc": {
```

Comment:
there's nothing inherently wrong with including nyc/mocha configuration here, but it does litter up the package.json somewhat.  I personally prefer separate files <https://github.com/istanbuljs/nyc#configuration-files> & <https://mochajs.org/#priorities>

Reply:
The one big benefit other than being better separation is that they both support .js files for commonjs which means you can have comments in them rather than having to stick to strict JSON which doesn't support comments :(

Reply:
Yeah... I'm rather the opposite. While the lack of comments does bug the crap out of me, having dozens of top-level files bugs me even more. I really like being able to have a single place to look for configuration, as opposed to having to look through file after file after file after...